javascript/javascript1/week4/voice-assistant.js

```diff
@@ -0,0 +1,204 @@
+let userName = "";
+let todoList = [];
+let todo = "";
+const today = new Date();
+const month = today.toLocaleDateString(undefined, { month: 'long' });
+
+// EXTRA FEATURE : VOICE ASSISTANT ANSWERS THESE INPUTS WITH ARRAY MEMBERS OF REPLY, ALTERNATIVE OR ROBOT 
+const input = [
+    //0 
+    ["hi", "hey", "hello"],
+    //1
+    ["how are you", "how are things"],
+    //2
+    ["what is going on", "what is up"],
+    //3
+    ["happy", "good", "well", "fantastic", "cool"],
+    //4
+    ["bad", "bored", "tired", "sad"],
+    //5
+    ["tell me story", "tell me joke"],
+    //6
+    ["thanks", "thank you"],
+    //7
+    ["bye", "good bye", "goodbye"],
+    //8
+    ["who are you", "what are you"],
+];
+
+const reply = [
+    //0 
+    ["Hello!", "Hi!", "Hey!", "Hi there!"],
+    //1
+    [
+        "Fine... how are you?",
+        "Pretty well, how are you?",
+        "Fantastic, how are you?"
+    ],
+    //2
+    [
+        "Nothing much",
+        "Exciting things!"
+    ],
+    //3
+    ["Glad to hear it"],
+    //4
+    ["Why?", "Cheer up buddy"],
+    //5
+    ["What about?", "Once upon a time..."],
+    //6
+    ["You're welcome", "No problem"],
+    //7
+    ["Goodbye", "See you later"],
+    //8
+    ["I am a voice assistant", "A voice assistant"],
+];
+
+const alternative = [
+    "Same",
+    "Go on...",
+    "Try again",
+    "I'm listening...",
+    "Bro..."
+];
+const robot = ["How do you do, fellow human", "I am not a bot"];
+// EXTRA FEATURE ENDS HERE
+
+function getReply(command) {
+    let newCommand = command.toLowerCase();
+    let commandArray = newCommand.split(" ");
+
+    if (newCommand.includes("hello my name is ")) {
+        if (userName === commandArray[commandArray.length - 1]) {
+            return `I know you already ${userName}`
+        } else {
+            userName = commandArray[commandArray.length - 1];
+            return `Nice to meet you ${userName}`;
+        }
+    }
+
+    else if (newCommand === "what is my name") {
```

Comment:
Remove the blank line. The `if` and `else if` are one structure so don't separate them.
```suggestion
    }
    else if (newCommand === "what is my name") {
```